addons/website_forum/static/src/js/website_forum.js

```diff
@@ -501,6 +510,57 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
         $('.forum_intro').slideUp();
         return true;
     },
+    /**
+     * @private
+     * @param {Event} ev
+     */
+    _onFlagValidatorClick: async function (ev) {
+        ev.preventDefault();
+        const currentTarget = ev.currentTarget;
+        const result = await this._rpc({
+            model: 'forum.post',
+            method: currentTarget.dataset.action,
+            args: [parseInt(currentTarget.dataset.postId)],
+        });
+        if (result) {
```

Comment:
How could result be falsy? Either it crashes and it handled at lower level (which shows a toaster) or it will just return something